types/blog.ts

```diff
@@ -4,6 +4,7 @@ export interface BlogPost {
   date: string;
   category: string;
   slug: string;
+  readingTime: string;
```

Comment:
We don't actually provide this from the sources of the blog post yet, right?

So maybe make this optional (nullable) and add a todo to verify in the future about the blog generation.

Reply:
Updated!